views/islandora_usage_stats_collections.views_default.inc

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Implements hook_views_default_views().
+ */
+function islandora_usage_stats_collection_views_default_views() {
```

Comment:
I added the view to the usage_stats as another array entry based on the islandora_batch.